courses/free-courses-hi.md

```diff
@@ -101,6 +102,12 @@
 * [Stack Playlist \| Interview Questions \| Coding \| Tutorials \| Data Structures](https://www.youtube.com/playlist?list=PL_z_8CaSLPWdeOezg68SKkeLN4-T_jNHd) - Aditya Verma
 
 
+### Databases
+
+* [DBMS(Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers (YouTube)
+* [DBMS(Database Management System)Placement Series](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp (YouTube)
```

Comment:
- Don't invent titles! Copy as it is
- YT playlist don't need notes
- Complete course instructors
 
```suggestion
* [DBMS (Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers, `tch.:` Varun Singla
* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp, `tch.:` Love Babbar, `tch.:` Lakshay Kumar
```

Reply:
Solved at 207d267d2397e99062e811126c72978b7c10016f